server/app.js

```diff
@@ -1,51 +1,86 @@
 const express = require('express');
 const db = require('./db-connection');
 require('dotenv').config();
-
+var cors = require('cors');
 const app = express();
 const port = 8000;
+app.use(cors());
+
+// Enable body parser
+app.use(express.json());
+
+
 
-app.get('/', (req, res) => {
-  res.send('Hello World');
-});
 
-//Example endpoint that sends a try-catch wrapped query to the database
-app.get('/test_db', async (req, res) => {
+app.post('/post_comment', async (req, res) => {
   try {
+    let time_posted = new Date();
+    let user_id = 3;
+    let comment_text = req.body.comment_text;
+    let proposal_id = 1;
```

Comment:
Make user_id and proposal_id also come as inputs from the request. User_id and proposal_id should be passed in as **query paramters** and you will be able to access them through `req.query.user+id`, etc.